src/components/dateline.tsx

```diff
@@ -23,7 +23,7 @@ const darkStyles = darkMode`
 
 const styles = css`
     ${textSans.xsmall()}
-    color: ${text.supporting};
+    color: ${neutral[20]};
```

Comment:
I think we just want this darker colour for opinion types and `text.supporting` for the others

Reply:
👍 will change - also UI tests seem to be stalled

Reply:
I think that's Chromatic picking up a UI change in the dateline component! can show you how to approve these once you've made the changes